wire-ios/Wire-iOS/Sources/UserInterface/MainController/MainTabBarController.swift

```diff
@@ -0,0 +1,94 @@
+        selectedImage: .init(resource: .contactsFilled)
+    )
+    mainTabBarController.viewControllers?[0].tabBarItem.accessibilityIdentifier = "bottomBarPlusButton"
+    mainTabBarController.viewControllers?[0].tabBarItem.accessibilityLabel = L10n.Accessibility.TabBar.Contacts.description
+    mainTabBarController.viewControllers?[0].tabBarItem.accessibilityHint = L10n.Accessibility.TabBar.Contacts.hint
+
+    mainTabBarController.viewControllers?[1].tabBarItem = .init(
+        title: L10n.Localizable.ConversationList.BottomBar.Conversations.title,
+        image: .init(resource: .TabBar.conversations),
+        selectedImage: .init(resource: .TabBar.conversationsFilled)
+    )
+    mainTabBarController.viewControllers?[1].tabBarItem.accessibilityIdentifier = "bottomBarRecentListButton"
+    mainTabBarController.viewControllers?[1].tabBarItem.accessibilityLabel = L10n.Accessibility.TabBar.Conversations.description
+
+    mainTabBarController.viewControllers?[2].tabBarItem = .init(
+        title: L10n.Localizable.ConversationList.BottomBar.Folders.title,
+        image: .init(resource: .foldersOutline),
+        selectedImage: .init(resource: .foldersFilled)
+    )
+    mainTabBarController.viewControllers?[2].tabBarItem.accessibilityIdentifier = "bottomBarFolderListButton"
+    mainTabBarController.viewControllers?[2].tabBarItem.accessibilityLabel = L10n.Accessibility.TabBar.Folders.description
+
+    mainTabBarController.viewControllers?[3].tabBarItem = .init(
+        title: L10n.Localizable.ConversationList.BottomBar.Archived.title,
+        image: .init(resource: .archiveOutline),
+        selectedImage: .init(resource: .archiveFilled)
+    )
+    mainTabBarController.viewControllers?[3].tabBarItem.accessibilityIdentifier = "bottomBarArchivedButton"
+    mainTabBarController.viewControllers?[3].tabBarItem.accessibilityLabel = L10n.Accessibility.TabBar.Archived.description
+    mainTabBarController.viewControllers?[3].tabBarItem.accessibilityHint = L10n.Accessibility.TabBar.Archived.hint
+
+    mainTabBarController.selectedIndex = 1
+    mainTabBarController.tabBar.backgroundColor = SemanticColors.View.backgroundDefault
+    mainTabBarController.tabBar.unselectedItemTintColor = SemanticColors.Label.textTabBar
+
+    return mainTabBarController
+}
+
+// MARK: - Previews
+
+struct MainTabBarController_Previews: PreviewProvider {
+
+    static var previews: some View {
+        MainTabBarControllerWrapper()
+            .ignoresSafeArea(edges: .all)
+    }
+}
+
+private struct MainTabBarControllerWrapper: UIViewControllerRepresentable {
+
+    func makeUIViewController(context: Context) -> UITabBarController {
+        let tabItem: (String) -> UIHostingController = { .init(rootView: Text($0)) }
+        return MainTabBarController(
+            contacts: tabItem("Contacts"),
+            conversations: tabItem("Conversations"),
+            folders: tabItem("Folders"),
+            archive: tabItem("Archive")
```

Comment:
Should those titles become localized?

Reply:
This is only for the Preview, but fixed.